wled00/cfg.cpp

```diff
@@ -18,6 +18,13 @@ bool deserializeConfig(JsonObject doc, bool fromFS) {
 
   //long vid = doc[F("vid")]; // 2010020
 
+  #ifdef WLED_USE_ETHERNET
```

Comment:
Per discussions with AirCookie and Blazoncek, initialization of ethernet should occur prior to other configuration (e.g., pins should be allocated to ethernet before being allocated to LEDs).